internal/servers/private_projects_server_test.go

```
@@ -362,4 +378,184 @@ var _ = Describe("Private projects server", func() {
+		Expect(err).To(HaveOccurred())
+		status, ok := grpcstatus.FromError(err)
+		Expect(ok).To(BeTrue())
+		Expect(status.Code()).To(Equal(grpccodes.InvalidArgument))
+		Expect(status.Message()).To(Equal(
+			"project 'my-parent' doesn't exist",
+		))
+	})
+
+	It("Can't create a project with more than 4 segments in the name", func() {
+		// Create nested projects to the maximun depth:
+		const max = 4
```

golangci-lint (2.12.2)

[error] const max has same name as predeclared identifier

(predeclared)